lib/req-res/requester.js

```diff
@@ -23,6 +23,13 @@ function Requester ( options ) {
 
 	if ( data instanceof Error ) {
 		this.error = data;
+	} else {
+		this.queue = mixins.getQueueName( options.messageType, 'req-res' );
+
+		// set error listener
+		this.on( 'error', ( errorReq ) => {
+			config.logger.error( `Queue: ${this.queue} -  ${errorReq.message}` );
```

Comment:
Created with the dummy logger, https://github.com/sinet/lapin/blob/590d675b600b9e5e19bd8f4e83202da220f8d4b9/lib/req-res/responder.js#L12